LicheeCluster4A/README.md

```diff
@@ -16,13 +16,13 @@
 - RevyOS
     - Download link: [here](https://mirror.iscas.ac.cn/revyos/extra/images/lpi4a/)
     - Reference Installation Document: [here](https://docs.revyos.dev/)
-- openKylin
+- openKylin 2.0 alpha
```

Comment:
For all oK 2.0 Alpha tests we should probably bump to the latest version...
But not for now, not this PR at least.